.sbtopts

```diff
@@ -0,0 +1,2 @@
+-Djavax.net.ssl.trustStore=/Users/toniogela/.config/PDNDTrustStore
```

Comment:
⚠️ remove from commit and update the .gitignore file

Reply:
yep, it was a test, I'll squash the commit to remove it from history too